solution-base/mongodb/charts/mongodb-sharded/templates/config-server/config-server-statefulset.yaml

```diff
@@ -208,8 +208,7 @@ spec:
           readinessProbe:
             exec:
               command:
-                - mongo
-                - --disableImplicitSessions
+                - mongosh
```

Comment:
All of this is coming from the official bitnami charts. We need to change them manually because we do not use helm in the Artesca cluster, to even if the kubernetes resources are "helm-managed", this is something we handle here.

here it's coming from https://github.com/bitnami/charts/blob/9f7ac81da075932c057a406970154f73bbdd33cd/bitnami/mongodb-sharded/templates/config-server/config-server-statefulset.yaml#L211

We only change them when we need specific configuration for our product, but for that we use patch files (from Alex's commits).